### ISSUE TITLE
gradient back propagation through image transforms

### ISSUE THREAD

**Issue by Yasinli1028** [tensorflow/tensorflow#17442]
As we know from the description of tf.contrib.image.transform(images, transforms): "Note that gradients are not backpropagated into transformation parameters".

Here, I would like to know whether gradients can be correctly backpropagated into images?

If yes, whether it is automatic?

Do I need to explicitly define the gradient using gradient_map_override?

Any suggestions would be helpful. Thanks

Have I written custom code: N/A
OS Platform and Distribution: Ubuntu 15.04
TensorFlow installed from: pip 
TensorFlow version: 1.6.0
Bazel version: N/A
CUDA/cuDNN version: 9.0
GPU model and memory: 24G Quadro P6000
Exact command to reproduce: N/A

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by tensorflowbutler** [comment]
Nagging Awaiting Response: It has been 14 days with no activityand the `awaiting response` label was assigned. Is this still an issue?

**Comment by xuyingxiao** [comment]
I have the same problem, have you solved? Or the gradient couldn't back propagation  into transformation parameters?
Any suggestions would be helpful. Thanks
